datahub/dataset/company_export_country_history/test/test_views.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from django.urls import reverse
+from freezegun import freeze_time
+
+from datahub.company.test.factories import CompanyExportCountryHistoryFactory
+from datahub.dataset.core.test import BaseDatasetViewTest
+
+
+def get_expected_data_from_export_country_history(export_country_history):
+    """Returns export_country_history data as a dictionary"""
+    return {
+        'id': str(export_country_history.id),
+        'company_id': str(export_country_history.company_id),
+        'country__name': export_country_history.country.name,
+        'country__iso_alpha2_code': export_country_history.country.iso_alpha2_code,
+        'history_date': export_country_history.history_date.strftime(
+            '%Y-%m-%dT%H:%M:%S.%fZ',
+        ),
```

Comment:
Would be good to use `format_date_or_datetime()` similar to the other data set tests:
```suggestion
        'history_date': format_date_or_datetime(export_country_history.history_date),
```

Reply:
updated, https://github.com/uktrade/data-hub-api/pull/2674/files#diff-a11b49b1f2ef4a012813a4368ba337a3R17